plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -335,7 +336,11 @@ public void launch(ILaunchConfiguration configuration, String mode, final ILaunc
 
     LocalAppEngineServerBehaviour serverBehaviour = (LocalAppEngineServerBehaviour) server
         .loadAdapter(LocalAppEngineServerBehaviour.class, null);
-
+    
+    IVMInstall vmInstall = getVMInstall(configuration);
```

Comment:
Or would we be better passing this information via the `startDevServer()` call at the end of this method?